scripts/integration/datadog-traces/compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: '3'
```

Comment:
I find it interesting that this is separate from the `agent` test above, when it should be the same component?

@neuronull are there reasons we can't reduce these to a single test setup?

Reply:
I'm not really clear on what the `version` field of the yaml file does. hmmm

But to the question of why there is a separate integration suite for `datadog-traces`, I think the main reason is that the agent service has different configurations for the datadog traces test. 

Also, we would need to have separate args to nextest passed for the source vs sink .

Reply:
Doh - sorry yeah, it wasn't specific to that version line 😂 

It seemed like the other test was doing all "sources and sinks" that are Datadog (except for this trace one), I may have misread it.

Reply:
also a side note partially for myself- on my TODO list still is to update the scope of the integration test files to only run the integration tests.
I'll wait until after this is merged in for that.

Reply:
Good plan, however most of these integration test configs do already run only a limited set of tests.

Reply:
> I'm not really clear on what the `version` field of the yaml file does. hmmm

The `version` field is the version of the compose standard to use.